cmd/run.go

```diff
@@ -390,12 +391,42 @@ func Run(isDebug *bool) *cli.Command {
 }
 
 func printErrorsInResults(errorsInTaskResults []*scheduler.TaskExecutionResult, s *scheduler.Scheduler) {
-	errorPrinter.Printf("\nFailed tasks: %d\n", len(errorsInTaskResults))
-	for _, t := range errorsInTaskResults {
-		errorPrinter.Printf("  - %s\n", t.Instance.GetHumanReadableDescription())
-		errorPrinter.Printf("    └── %s\n\n", t.Error.Error())
+	errorPrinter.Printf("\nFailed assets: %d\n", len(errorsInTaskResults))
```

Comment:
this will still print `5`, we want to print `1`. There might be one asset execution, resulting in 5 tasks being executed (1 main + 4 quality checks), it currently prints 5, we wanna print 1